SUMMARY.md

```diff
@@ -9,15 +9,16 @@
    * [Addresses](cli/addresses.md)
        * [Add a prefix](cli/add_a_prefix.md)
        * [View a prefix](cli/view_a_prefix.md)
-       * [Search for prefixes](cli/search_for_prefixes.md)
+       * [List prefixes](cli/list_prefixes.md)
```

Comment:
I think it's better to call it "search" :)
bla bla, discussion on IRC.. it was really a mistake on our part to call the function "list" in the first place IMHO